src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -266,10 +266,9 @@ std::map<std::tuple<AccountID, AccountID, Currency>, STAmount>
 PaymentSandbox::balanceChanges (ReadView const& view) const
 {
     using key = std::tuple<AccountID, AccountID, Currency>;
-    // Map of delta trust lines
-    // As a special case, when both ends of the trust line are the same currency, then
-    // it's delta currency for that issuer
-    // To get the change in XRP balance, Account == root, issuer == root, currency == XRP
+    // Map of delta trust lines. As a special case, when both ends of the trust
+    // line are the same currency, then it's delta currency for that issuer. To
+    // get the change in XRP balance, Account == root, issuer == root, currency == XRP
     std::map<key, STAmount> result;
 
     // populate a dictionary with low/high/currency/delta // this can be
```

Comment:
The re-wrapping of the previous paragraph is a good thing.  But it's actually this following paragraph that has a couple of "`//`" in the middle of the paragraph that I suspect may be misplaced.